Documentation/Developer-Guide.md

```diff
@@ -0,0 +1,35 @@
+Intro to WCF
+================
+
+Windows Communication Foundation (WCF) has been a part of the full .NET framework for years.  This WCF repository contains source code for all the client libraries originally available for the Windows Store but made compatible with the new [.NET Core Runtime](https://github.com/dotnet/coreclr).
+
+Project Goals and Priorities
+============================
+
+Microsoft published WCF to GitHub, with the following goals:
+
+- Establish a high-quality open source .NET implementation of the WCF client libraries.
+- Work with the open source community to manage and extend the available libraries.
+
+Reporting Issues
+================
+To report an issue or just to open the discussion for a proposed feature, open a [New Issue](https://github.com/dotnet/wcf/issues/new). Please assign labels to associate it with the appropriate library if you know it. Example: "System.ServiceModel.Http".
+
+Contributing
+============
+
+Please read [Contributing](https://github.com/dotnet/corefx/blob/master/Documentation/contributing.md) to .NET Core before making your first contribution.
+
```

Comment:
The link here should probably now point to our own Contributing.md.  Note: CoreFx went to all lowercase names for the file names.  I don't know why but it might be a convention to consider.